src/App.tsx

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css'
 import TodoList from "./TodoList.tsx";
 import AddTodoForm from "./AddTodoForm.tsx";
 
 function App() {
-
+    const [newTodo, setNewTodo] = useState<string>();
```

Comment:
nit: if type `string` could initialize as empty string. if this causes display issues at the template level, then there's space for conditionally displaying that `p` tag